android/app-newm/build.gradle.kts

```diff
@@ -86,20 +86,10 @@ android {
 
 dependencies {
 
-//    kapt(Airbnb.showkaseProcessor)
-    implementation(platform(Google.firebase))
-    implementation(project(Modules.coreTheme))
-    implementation(project(Modules.coreResources))
-    implementation(project(Modules.coreUiUtils))
-    implementation(project(Modules.login))
-    implementation(project(Modules.musicPlayer))
-    implementation(project(Modules.barcodeScanner))
-    implementation(project(Modules.shared))
-
+    implementation("com.jakewharton:process-phoenix:2.1.2")
```

Comment:
Why not using the a dependency object string here?

Reply:
Need to do a dependency cleaning soon, will create a ticket to take care of that and a few others 👍🏼